products/BellHybrid/apps/common/include/common/models/SongsModel.hpp

```diff
@@ -3,44 +3,44 @@
 
 #pragma once
 
-#include "data/RelaxationCommon.hpp"
 #include <common/SoundsRepository.hpp>
+#include <common/widgets/LabelListItem.hpp>
 #include <apps-common/models/SongsModelInterface.hpp>
 #include <gui/widgets/ListItemProvider.hpp>
 
-namespace app::relaxation
+namespace app
 {
 
-    class RelaxationSongsProvider : public app::DatabaseModel<db::multimedia_files::MultimediaFilesRecord>,
-                                    public gui::ListItemProvider
+    class SongsProvider : public app::DatabaseModel<db::multimedia_files::MultimediaFilesRecord>,
+                          public gui::ListItemProvider
     {
       public:
-        virtual ~RelaxationSongsProvider() = default;
+        virtual ~SongsProvider() = default;
         using OnActivateCallback =
             std::function<bool(const db::multimedia_files::MultimediaFilesRecord &selectedSound)>;
-        explicit RelaxationSongsProvider(ApplicationCommon *application);
+        explicit SongsProvider(ApplicationCommon *application);
         virtual void createData(OnActivateCallback activateCallback) = 0;
     };
 
-    class RelaxationSongsModel : public RelaxationSongsProvider
+    class SongsModel : public SongsProvider
     {
       private:
         ApplicationCommon *application;
         std::unique_ptr<AbstractSoundsRepository> songsRepository;
-        std::map<MusicType, std::string> pathPrefixes;
+        std::map<std::string, std::string> pathPrefixes;
```

Comment:
I'd think about adding some `using`s to typedef these strings to some more self-explanatory types - I'm not very familiar with this part of GUI and at a first glance it's quite hard to determine what this map contains when considering it from high-level (implementational) point of view

Reply:
I created an alias by `using` for this map. Done ;)